analyzer-models/src/test/java/ffb/test/utilities/GenericTests.java

```diff
@@ -0,0 +1,52 @@
+package ffb.test.utilities;
+
+import ffb.analyzer.models.espn.TransactionCounter;
+import org.junit.Assert;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class GenericTests {
+    public static void validateGetMethodsReturnNonNullValue(List<?> testObjects) throws InvocationTargetException, IllegalAccessException {
+        List<Method> getterMethods = new ArrayList<Method>();
+
+        for (Method method : testObjects.get(0).getClass().getMethods()) {
+            if (isGetterMethod(method)) {
+                getterMethods.add(method);
+            }
+        }
+
+        for (var i = 0; i < testObjects.size(); i++) {
+            for (var j = 0; j < getterMethods.size(); j++) {
+                var method = getterMethods.get(j);
+//                Assert.assertNotNull(getterMethods.get(j).invoke(testObjects.get(i)));
+                Assert.assertNotNull(method.invoke(testObjects.get(i)));
+            }
+        }
+    }
+
+    public static void validateGetMethodsReturnNonNullValue(Object testObject) throws InvocationTargetException, IllegalAccessException {
+        for (Method method : testObject.getClass().getMethods()) {
```

Comment:
How about:

```java
testObject.getClass().getMethods().forEach(method -> method.invoke(testObject));
```

Reply:
Same minor tweaks as before, and Intellij wasn't happy without a try/catch.